en/sql/tuning.rst

```diff
@@ -498,6 +509,15 @@ The following are the explanation regarding items of trace statistics.
     *   readkeys: the number of the keys which are read in btree when this operation is performed
     *   filteredkeys: the number of the keys to which the key filter is applied from the read keys
     *   rows: the number of result rows when this operation is performed; the number of result rows to which key filter is applied
+
+*   temp: data scanning job with temp file
+
+    *   hash: hash list scan or not. See :ref:`NO_HASH_LIST_SCAN <no-hash-list-scan>` hint.
+    *   buildtime: the estimated time(ms) in building hash table.
+    *   time: the estimated time(ms) in probing hash table.
+    *   fetch, ioread: page fetching count and I/O read count in the temp file of this operation
+    *   readrows: the number of read rows when this operation is performed
+    *   rows: the number of result rows when this operation is performed
```

Comment:
readrows: the number of  read rows --> readrows: the number of rows read

Reply:
I fixed it.